src/controllers/user.controller.js

```diff
@@ -0,0 +1,96 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../modules/User.js';
+import { userService } from '../service/user.service.js';
+import bcrypt from 'bcrypt';
+
+const getAllActivated = async (req, res) => {
+  const users = await userService.getAllActivated();
+
+  res.send(users.map(userService.normalize));
+};
+
+const changeName = async (req, res) => {
+  const { userName } = req.params;
+  const { newName } = req.body;
+
+  if (!userName || !newName) {
+    throw ApiError.badRequest('Something wrong');
+  }
+
+  await userService.changeName(userName, newName);
+  res.send('Name change sucssessfull');
+};
+
+const changePassword = async (req, res) => {
+  const { userName } = req.params;
+  const { oldPassword, newPassword, confirmation } = req.body;
+
+  const user = await User.findOne({ where: { userName } });
+
+  const isPasswordValid = await bcrypt.compare(oldPassword, user.password);
+
+  if (!isPasswordValid) {
+    throw ApiError.badRequest('Wrong old password');
+  }
+
+  if (!userName) {
+    throw ApiError.badRequest('Something wrong');
+  }
+
+  if (newPassword !== confirmation) {
+    res.status(400).send('Passwords do not match');
+
+    return;
+  }
+
+  const hashPass = await bcrypt.hash(newPassword, 10);
+
+  await userService.changePassword(userName, hashPass);
+  res.send('Password change sucssessfull');
+};
+
+const changeEmail = async (req, res) => {
+  const { userName } = req.params;
+  const { password, newEmail } = req.body;
+
+  const user = await User.findOne({ where: { userName } });
```

Comment:
Consider handling potential exceptions that might occur during the database query operation to improve robustness.